### PR TITLE
adapt rust code of conduct as base for aframe code of conduct (fixes #954)

### LINKED ISSUE #954: Add Code of Conduct

### PR DESCRIPTION
**Pull request by ngokevin**
**Description:**

Use the Rust Code of Conduct as a base for A-Frame's Code of Conduct. This can be expanded upon or revised as our community grows. Fortunately, so far, the A-Frame community has been extremely welcoming and friendly. But good to have this in place.

Eventually, we may want this on the website, but let's start here.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+Adapted from the [Rust Code of Conduct](https://www.rust-lang.org/en-US/conduct.html).
+
+## Conduct
+
+- We are committed to providing a friendly, safe and welcoming environment for
+  all, regardless of level of experience, gender, gender identity and expression,
+  sexual orientation, disability, personal appearance, body size, race,
+  ethnicity, age, religion, nationality, or other similar characteristic.
+- On IRC, please avoid using overtly sexual nicknames or other nicknames that
```
Comment (dmarcos) [inline CODE_OF_CONDUCT.md]: We don't use IRC

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- Remarks that moderators find inappropriate, whether listed in the code of
+  conduct or not, are also not allowed.
+- Moderators will first respond to such remarks with a warning.
+- If the warning is unheeded, the user will be kicked, i.e., kicked out of the
+  communication channel to cool off.
+- If the user comes back and continues to make trouble, they will be banned,
+  i.e., indefinitely excluded.
+- Moderators may choose at their discretion to un-ban the user if it was a
+  first offense and they offer the offended party a genuine apology.
+- If a moderator bans someone and you think it was unjustified, please take it
```
Comment (dmarcos) [inline CODE_OF_CONDUCT.md]: We don't have a way to ban people that I know. Does this apply?

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+## Moderation
+
+These are the policies for upholding our community's standards of conduct. If
+you feel that a thread needs moderation, please contact one of the A-Frame team
+members.
+
+- Remarks that violate the A-Frame standards of conduct, including hateful,
+  hurtful, oppressive, or exclusionary remarks, are not allowed. (Cursing is
+  allowed, but never targeting another user, and never in a hateful manner.)
+- Remarks that moderators find inappropriate, whether listed in the code of
```
Comment (dmarcos) [inline CODE_OF_CONDUCT.md]: Do we have moderators?

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (ngokevin) [inline CODE_OF_CONDUCT.md]: technically we can as admins on slack

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (ngokevin) [inline CODE_OF_CONDUCT.md]: i put us down as the initial/interim list and put out a call for more.

**Pull request merged**